server/libs/infra/src/entities/user.entity.ts

```diff
@@ -27,6 +27,9 @@ export class UserEntity {
   @Column({ unique: true })
   email!: string;
 
+  @Column({ type: 'varchar', unique: true, default: null })
```

Comment:
Unique constraint violations will result in a non-descriptive internal server error. Not a huge issue, but a more useful error response would be nice

Reply:
Added a check for uniqueness, similar to email.